utils/auth/provider.tsx

```diff
@@ -62,12 +72,15 @@ const AuthProvider = ({ children }: PropsWithChildren) => {
         signin,
         signup,
         signout,
+        oAuthGoogleSignin,
         accessToken,
         refreshToken,
         authenticated,
       }}
     >
-      {children}
+      <GoogleOAuthProvider clientId="43480745913-1lhqt43oe2gir7fqdv5058m5l3q1hl2p.apps.googleusercontent.com">
```

Comment:
Let's add this as a `.env` config variable, read the guide [here](https://nextjs.org/docs/basic-features/environment-variables#exposing-environment-variables-to-the-browser)

Reply:
done